src/modules/visits/components/VisitRequestsOverview/VisitRequestsOverview.tsx

```diff
@@ -181,21 +176,16 @@ const VisitRequestOverview: FC<VisitRequestOverviewProps> = ({ columns }) => {
 		} else {
 			return (
 				<div className="l-container l-container--edgeless-to-lg">
-					<Table
+					<Table<Visit>
```

Comment:
Looks like a typo

Reply:
Nope, this is the way you use typescript generics with react components:
https://www.developerway.com/posts/typescript-generics-for-react-developers

![image](https://user-images.githubusercontent.com/1710840/191781079-7cf4348d-9ba6-4a55-8a3a-7899300720f2.png)